src/lib/Witch/Instances.hs

```diff
@@ -1015,6 +1015,14 @@ instance From.From ByteString.ByteString ShortByteString.ShortByteString where
 instance TryFrom.TryFrom ByteString.ByteString Text.Text where
   tryFrom = Utility.eitherTryFrom Text.decodeUtf8'
 
+-- | Converts via 'Text.Text'.
+instance TryFrom.TryFrom ByteString.ByteString LazyText.Text where
+  tryFrom = Utility.eitherTryFrom $ fmap (Utility.into @LazyText.Text) . Utility.tryInto @Text.Text
+
+-- | Converts via 'Text.Text'.
+instance TryFrom.TryFrom ByteString.ByteString String where
+  tryFrom = Utility.eitherTryFrom $ fmap (Utility.into @String) . Utility.tryInto @Text.Text
```

Comment:
Instances like this are convenient to write, but the error gets a little confusing:

``` hs
ghci> Witch.TryFrom.tryFrom @Data.ByteString.ByteString @String (Data.ByteString.pack [0xff])
Left (TryFromException @ByteString @[Char] "\255" (Just (TryFromException @ByteString @Text "\255" (Just Cannot decode byte '\xff': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream))))
```

Reply:
... as noted here: https://github.com/tfausak/witch/pull/32#discussion_r641788442